index.rst

```diff
@@ -372,30 +372,41 @@ c.l.o stubb
 
   Here is where we currently are in the release process. Current step in bold.
 
-  Precursor Steps
+  Release Precursor Steps
   ---------------------------------
 
   1. Identify any pre-release blockers ("must-have features") :tools:
-  2. Wait for them to clear
+  Contributors check if there are outstanding issues that have to be included in the next M.m.p release and related them as blocker to the above issue DM-XXXXX.
+  1. Create Jira issues for each release activity.
+
+  Release Jira issue: https://jira.lsstcorp.org/browse/DM-XXXXX
+  Tentative weekly to use as starting point for the release is w_20YY_WW
+  Tentative target date to close the release is YYYY-MM-DD.
 
   Release Engineering Steps
   -------------------------------
 
-  1. Branch 888.0.x of newinstall.sh
-  1. Build and publish rc1 release candidate (based on w.9999.52)
-  1. **Wait for first round of bugs to clear**
-  1. Build and publish additional rcX releases if/as necessary
-  1. Full OS testing (see https://ls.st/faq )
-  1. Build and publish final release
+  1. Check that the weekly build is scientifically suitable to be used as starting point for the release
```

Comment:
This seems more like a precursor than an engineering step.

Reply:
Agreed. I move it above.